src/development/loadbalancer/nginx.conf

```diff
@@ -0,0 +1,44 @@
+worker_processes 1;
+
+events { worker_connections 1024; }
+
+http {
+
+    client_max_body_size 50M;
+
+    sendfile on;
+	
+	upstream localtest {
+        server host.docker.internal:5101;
+    }
+
+    upstream app {
+        server host.docker.internal:5005;
+    }
+
+    server {
+		listen 80;
+        server_name altinn3local.no localhost;
+
+        proxy_redirect      off;
+        proxy_set_header    Host $host;
+        proxy_set_header    X-Real-IP $remote_addr;
+        proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
+		
+		
+		location = / {
+        proxy_pass          http://localtest/Home/;
+		}		
+		
+		rewrite ^/.*/.*/staticfiles(.*)$ $1 last;
```

Comment:
this line could be removed as we no longer require the staticfiles redirect

Reply:
Removed this line